imspinner.h

```diff
@@ -1478,7 +1479,7 @@ namespace ImSpinner
       for (size_t arc_num = 0; arc_num < arcs; ++arc_num) {
           window->DrawList->PathClear();
           float arc_start = 2 * IM_PI / arcs;
-          float b = ease((ease_mode)mode, start + arc_num * PI_DIV(2) / arcs, IM_PI, 1, 0);
+          float b = ease((ease_mode)mode, start + static_cast<float>(arc_num) * PI_DIV(2) / static_cast<float>(arcs), IM_PI, 1.0f, 0.0f);
```

Comment:
static_cast seems not need here, PI_DIV is float, so arc_num also will be float in result

Reply:
Removed the instances of `static_cast`